packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -410,13 +416,31 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
   }
 
   Widget _buildChildren(BuildContext context, Widget? child) {
+    final ThemeData theme = Theme.of(context);
     final ExpansionTileThemeData expansionTileTheme = ExpansionTileTheme.of(context);
     final ShapeBorder expansionTileBorder = _border.value ?? const Border(
             top: BorderSide(color: Colors.transparent),
             bottom: BorderSide(color: Colors.transparent),
           );
     final Clip clipBehavior = widget.clipBehavior ?? expansionTileTheme.clipBehavior ?? Clip.none;
-
+    final MaterialLocalizations localizations = MaterialLocalizations.of(context);
+    final String onTapHint = _isExpanded
+      ? localizations.expansionTileExpandedTapHint
+      : localizations.expansionTileCollapsedTapHint;
+    String? semanticsHint;
+    switch (theme.platform) {
+      case TargetPlatform.iOS:
+      case TargetPlatform.macOS:
+        semanticsHint = _isExpanded
+          ? '${localizations.collapsedHint} double tap to ${localizations.expandedIconTapHint}'
```

Comment:
does the `double tap to` need to be localized?

Reply:
There is a "double tap to" on Android when running expansion tile example but I can't any localized string for it so I just hard-coded it here

See
https://github.com/flutter/flutter/blob/cb123947dfe457e7e26717672dbb746de018d9fe/packages/flutter/lib/src/gestures/multitap.dart#L378

Reply:
that debugDescrption is fine since it is test only. This string is customer facing, so I think we will need to localize it.

Reply:
Thank you! Added a localized string.